starter_scripts/push_docker_runner_image.sh

```diff
@@ -3,9 +3,9 @@
 set -euo pipefail
 
 if [ -n "${1-}" ]; then
-  image_suffix="$1"
```

Comment:
Please add a comment that we allow multiple suffixes, and we build only ones and retag mutliple times